list.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'enumerable'
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+
+  def each
+    yield @list
+  end
+end
+list1 = MyList.new(1, 2, 3, 4)
+
+# Test #all?
+puts(list1.all? { |e| e < 5 })
+puts(list.all? { |e| e > 5 })
+
+# Test #any?
+puts(list1.any? { |e| e == 2 })
+puts(list.any? { |e| e == 5 })
+
+# Test #filter
+puts list1.filter(&:even?)
```

Comment:
- [ ] [Optional] You initialized the list variable with the name `list1` But you are using `list` in the 2nd and 4th tests. If you will run the file, It will produce an error. I would suggest using the correct name while calling the variable.

![image](https://user-images.githubusercontent.com/83803414/209148218-40698f85-55be-4702-9c3c-6b5c0bfe3a66.png)

Reply:
Thank you. Its now resolved